panic: runtime error: invalid memory address during copy

Compiled master from source to have support for API v4
Testrun was fine, but with -y I got the following runtime error:

Any help would be welcome !

```
schwinn@belpc077:~/git/src/github.com/gotsunami/gitlab-copy/bin$ ./gitlab-copy -y gitlab.yml
source: aschwinn/silecs at https://gitlab.cern.ch/api/v4
target: al.schwinn/silecs at https://gitlab.com/api/v4
--
source: finding issues ... OK
source: 46 issues (26 opened, 20 closed)
source: 3 milestone(s): 3.0.0, 2.1.2, 2.1.0
source: 7 label(s): mergeWithCERN, silecs-cli-client, silecs-codegen, silecs-communication-cpp, silecs-diagnostic-cpp, silecs-model, virtual-controller
source: 237 notes                                                  
--
Copying labels ...
Copying issues ...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x729b45]

goroutine 1 [running]:
main.(*migration).migrateIssue(0xc4200892f0, 0x1, 0x866901, 0xc42022d560)
	/home/schwinn/git/src/github.com/gotsunami/gitlab-copy/cmd/gitlab-copy/migration.go:138 +0x3c5
main.(*migration).migrate(0xc4200892f0, 0x1, 0x1)
	/home/schwinn/git/src/github.com/gotsunami/gitlab-copy/cmd/gitlab-copy/migration.go:375 +0xf22
main.main()
	/home/schwinn/git/src/github.com/gotsunami/gitlab-copy/cmd/gitlab-copy/main.go:156 +0x933

```

--- a/cmd/gitlab-copy/migration.go
+++ b/cmd/gitlab-copy/migration.go
@@ -135,7 +135,7 @@ func (m *migration) migrateIssue(issueID int) error {
 			return fmt.Errorf("target: error fetching users: %s", err.Error())
 		}
 	}
-	if issue.Milestone.Title != "" {
+	if issue.Milestone != nil && issue.Milestone.Title != "" {
 		miles, _, err := target.Milestones.ListMilestones(tarProjectID, nil)
 		if err == nil {
 			found := false

